torch/fx/passes/runtime_assert.py

```diff
@@ -85,16 +271,41 @@ def insert_deferred_runtime_asserts(
         free_symbols,
         InnerTensorKey,
     )
-    from torch.utils._sympy.interp import sympy_interp
-    from torch.utils._sympy.numbers import int_oo
-    from torch.utils._sympy.reference import PythonReferenceAnalysis
 
-    # TODO: Request simplification on runtime asserts before emitting them
+    RUNTIME_ASSERT_OPS = [
+        torch.ops.aten._assert_async.msg,
```

Comment:
This is a kind of strange thing to handle here, since assert async operates on Tensor and we really shouldn't be touching Tensor compute in this pass

Reply:
I see, we'd been using it for the previous runtime asserts pass which we're deprecating, so I thought it was basically assert_scalar.

Reply:
Ah yes, if you wanted to peephole "scalar compute -> make a tensor -> assert async" that's... reasonable, I guess. Better not to generate code like that in the first place though.